static/logos/tech/datadog.svg

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should consider optimizing all the images we add. Ideally, we'd hook up automation to do it automatically for all existing & new images. Of course, we could also manually optimize these when adding the files, e.g. running `.svg` files through https://jakearchibald.github.io/svgomg/.

Reply:
I want to just get these out as a first set of work - I believe Luke has suggested we look at the content around these afterwards. This can 100% be on the followup issue to make sure these are better quality